yarn.lock

```diff
@@ -3358,7 +3358,7 @@ minimatch@^2.0.3:
 
 minimatch@^3.0.4:
   version "3.0.4"
-  resolved "https://registry.yarnpkg.com/minimatch/-/minimatch-3.0.4.tgz#5166e286457f03306064be5497e8dbb0c3d32083"
+  resolved "https://registry.npmjs.org/minimatch/-/minimatch-3.0.4.tgz#5166e286457f03306064be5497e8dbb0c3d32083"
```

Comment:
Unintentional?

Reply:
`yarn` seems to have a bug where it occasionally alternates between registries on some packages; not sure what causes it, but I've seen it once in a while

Reply:
I just mean we can safely omit this change from a PR.

Reply:
ok, I'll revert it.